website/static/js/osfHelpers.js

```diff
@@ -701,7 +701,22 @@ var confirmDangerousAction = function (options) {
 
     bootbox.dialog(bootboxOptions);
 };
-
+/**
+ * Maps an object to an array of {key: KEY, value: VALUE} pairs
+ *
+ * @param {Object} obj
+ * @returns {Array} array of key, value pairs
+ **/
+var iterObject = function(obj) {
```

Comment:
The function is used [here](https://github.com/barbour-em/osf.io/blob/feature/prereg-prize-lyndsy/website/templates/project/registration_editor_templates.mako#L50)